tests/test_api.py

```diff
@@ -1311,7 +1343,7 @@ class TestDogsAPI(APITestCase):
     def setUp(self):
         self.fixture = create_fixture()
 
-    def test_sort(self):
+    def test_sort_implied_all(self):
```

Comment:
I renamed a few of these. Happy to undo the name changes if you don't like them.

Reply:
Thanks for doing that!